audition/metric_directionality.py

```diff
@@ -0,0 +1,41 @@
+from catwalk.evaluation import ModelEvaluator
+import logging
+
+
+def greater_is_better(metric):
+    if metric in ModelEvaluator.available_metrics:
+        return ModelEvaluator.available_metrics[metric].greater_is_better
+    else:
+        logging.warning(
+            'Metric %s not found in available metrics, assuming greater is better',
+            metric
+        )
+        return True
+
+
+def sql_rank_order(metric):
+    if greater_is_better(metric):
+        return 'desc'
+    else:
+        return 'asc'
+
+
+def past_threshold(metric, df, threshold):
+    if greater_is_better(metric):
+        return df[df['raw_value'] >= threshold]
+    else:
+        return df[df['raw_value'] <= threshold]
+
+
+def best_in_dataframe(metric, df):
+    if greater_is_better(metric):
+        return df['raw_value'].max()
+    else:
+        return df['raw_value'].min()
+
+
+def idxbest(metric, df):
+    if greater_is_better(metric):
+        return df.idxmax()
```

Comment:
hmmm... or maybe this works because we're generally applying it after grouping by, in which case it's actually acting on a series rather than a dataframe, so I might just change the name of the argument from `df` to `series` (or something like that) to make it a little more clear...